locale/en/manageiq.po

```diff
@@ -72547,14 +72779,23 @@ msgid "The Template is not connected to an active Provider"
 msgstr ""
 
 #:
-#: ../app/models/manageiq/providers/awx/automation_manager/azure_credential.rb
+#: ../app/models/manageiq/providers/embedded_terraform/automation_manager/azure_credential.rb
 msgid "The Tenant ID for the Microsoft Azure account"
 msgstr ""
 
 #: ../app/controllers/ems_infra_controller.rb
 msgid "The URL is blank or not a String"
 msgstr ""
 
+#:
+#: ../app/models/manageiq/providers/embedded_terraform/automation_manager/ibm_cloud_credential.rb
+msgid ""
+"The User Name for IBM Cloud Classic Infrastructure. A value for this field is "
+"required when using classic IBM Cloud resources. A valid connection must have "
+"value for IBM Cloud API Key or, IBM Cloud Classic Infrastructure User Name and"
+" IBM Cloud Classic Infrastructure API Key."
+msgstr ""
```

Comment:
simplified in https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/21